pkg/internal/common/httphandler_test.go

```diff
@@ -4,14 +4,16 @@ Copyright SecureKey Technologies Inc. All Rights Reserved.
 SPDX-License-Identifier: Apache-2.0
 */
 
-package support
```

Comment:
It is unusual having a package not named after the directory it's in.

Reply:
updated